frontend/src/views/producer/dashboard/edition-form/CoverImage.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled, { withTheme } from 'styled-components';
+import { InputField } from '../../../../components/form';
+import Button from '../../../../components/button';
+
+const Image = styled.img`
+  width: 100%;
+  height: auto;
+  margin-bottom: 2rem;
+  border-radius: 8px;
+`;
+const FlexBox = styled.div`
+  display: flex;
+  width: 100%;
+  align-items: center;
+`;
+
+class CoverImage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      confirmedSrc: props.value,
+    };
+  }
+  onConfirm = () => {
+    this.setState({ confirmedSrc: this.props.value });
+  };
+  render() {
+    const { inputBackgroundColor, value, onChange } = this.props;
+    return (
+      <div>
+        <Image src={this.state.confirmedSrc} />
+        <FlexBox>
+          <InputField
+            style={{ width: '100%', marginRight: '1rem' }}
```

Comment:
Not sure if feasible here.. but in general I think it would be better to put as much of styles as possible into a separate css file instead of using inline styles.. that way it would become easier to update the theme of the site later.

Reply:
Usually that is applicable to really specific style (style-defining features). You can clearly see that most style live in its styled-components base.

In rare case, when you need a very contextual override of style, I think it's appropriate to inline. This InputField here has a very situational style for layout-ing, and thus should be appropriate.